app/frontend/src/theme.ts

```diff
@@ -35,6 +44,28 @@ const themeOptions: ThemeOptions = {
   },
   shape: {
     borderRadius,
+    navPaddingDesktop: spacing(10),
+    navPaddingMobile: spacing(7),
+    scrollBar: {
+      paddingInlineEnd: spacing(1),
+      overflow: "auto",
+      scrollbarWidth: "thin",
+      "&::-webkit-scrollbar": {
+        width: "0.5rem",
+        background: "rgba(0,0,0,0)",
+      },
+      "&::-webkit-scrollbar:hover": {
+        width: "0.5rem",
+        background: "rgba(0,0,0,0.1)",
+      },
+      "&::-webkit-scrollbar-thumb": {
+        borderRadius: "20px",
+        background: "rgba(0,0,0,0.2)",
+      },
+      "&::-webkit-scrollbar-thumb:hover": {
+        background: "rgba(0,0,0,0.3)",
+      },
```

Comment:
Does that mean it'll still look ugly on Firefox since this is WebKit specific CSS? 😢

Reply:
yes, but firefox supports `scrollbarWidth` at least